packages/select/src/components/select/suggest.tsx

```diff
@@ -38,6 +38,12 @@ export interface ISuggestProps<T> extends IListItemsProps<T> {
     /** Custom renderer to transform an item into a string for the input value. */
     inputValueRenderer: (item: T) => string;
 
+    /**
+     * The default selected item, or undefined. If provided, the item will be
```

Comment:
remove "or undefined" because the user is not expected to pass `defaultSelectedItem={undefined}`--they just omit the prop entirely in that case.

Reply:
actually, how about:

> The uncontrolled default selected item. This prop is ignored if `selectedItem` is used to control the state.